spec/requests/items_spec.rb

```diff
@@ -47,7 +47,18 @@
   describe 'GET /items/:id' do
     context 'when the record exists' do
       it 'returns the item' do
-        items = FactoryBot.create_list(:item, 10)
+        items = [
```

Comment:
Question: Why do you need so many items? Why not just one?

Reply:
Good point.